src/messenger.rs

```diff
@@ -315,6 +318,19 @@ where
     }
 
     pub async fn request<R>(&self, msg: R) -> Result<R::ResponseBody, RequestError>
```

Comment:
While this method is technically public, I doubt that anybody really uses it. So I think `request` should just return the metadata as well, in the same way that `request_with_version_ranges` does. This keeps the API kinda lean and avoids having all sorts of permutations.

Reply:
Resolved in [a12992f](https://github.com/influxdata/rskafka/pull/276/commits/a12992f6d3c7da192b0399e3462b89f7188b2758)